src/main/proto/stremio/core/deep_links/deep_links.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
+
+package stremio.core.deep_links;
+
+option java_package = "com.stremio.core.deep_links";
+
+message DiscoverDeepLinks {
+  required string discover = 1;
+}
+
+message LibraryDeepLinks {
```

Comment:
Not used?

Reply:
I mean no other message uses this struct, so it's not used or created anywhere?

Reply:
But it's not used in any other structure so it won't be possible to access it in kotlin?

Reply:
It can be accessed in kotlin, see this https://github.com/Stremio/stremio-android-tv/pull/179/files#diff-0c0b33f54972c0f4add22470f968390603a4d69023c35aee09df9b22877e66f8R4

Reply:
So, if it's a static value used only in kotlin I don't think we need a structure for it in proto, you can just have a s static string deeplink value. Proto is only needed for translation between rust and kotlin.